webapp/layouts/default.vue

```diff
@@ -1,62 +1,61 @@
 <template>
   <div>
-    <Nuxt />
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <NuxtLink class="navbar-brand" to="/">
+        Felni Hacker News
+      </NuxtLink>
+      <button
+        class="navbar-toggler"
+        type="button"
+        data-toggle="collapse"
+        data-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon" />
+      </button>
+
+      <div id="navbarSupportedContent" class="collapse navbar-collapse">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <NuxtLink class="nav-link" to="/">
+              News
+            </NuxtLink>
+          </li>
+        </ul>
+        <NavbarLoginButton :logged-out="loggedOut" @updateloggedoutstate="handleupdateloggedoutstate" />
+      </div>
+    </nav>
+    <div class="container">
+      <NewsList :logged-out="loggedOut" />
+    </div>
   </div>
 </template>
 
-<style>
-html {
-  font-family:
-    'Source Sans Pro',
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    Roboto,
-    'Helvetica Neue',
-    Arial,
-    sans-serif;
-  font-size: 16px;
-  word-spacing: 1px;
-  -ms-text-size-adjust: 100%;
-  -webkit-text-size-adjust: 100%;
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
-  box-sizing: border-box;
-}
-
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-}
-
-.button--green {
-  display: inline-block;
-  border-radius: 4px;
-  border: 1px solid #3b8070;
-  color: #3b8070;
-  text-decoration: none;
-  padding: 10px 30px;
-}
+<style src="bootstrap/dist/css/bootstrap.css">
+</style>
 
-.button--green:hover {
-  color: #fff;
-  background-color: #3b8070;
-}
+<script>
+import NavbarLoginButton from '../components/NavbarLoginButton/NavbarLoginButton'
 
-.button--grey {
-  display: inline-block;
-  border-radius: 4px;
-  border: 1px solid #35495e;
-  color: #35495e;
-  text-decoration: none;
-  padding: 10px 30px;
-  margin-left: 15px;
+export default {
+  name: 'Login',
+  components: {
+    NavbarLoginButton
+  },
+  data: () => {
+    return {
+      loggedOut: true
+    }
+  },
+  mounted () {
+    this.loggedOut = this.$apolloHelpers.getToken() === undefined
```

Comment:
There is a better way to implement this. I think I showed you in the seminar how to read the cookie on the server and initialize the store based on the token.